app/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_API_VERSION=v1
+REACT_APP_API_LOCAL_SERVER_PORT=8080
```

Comment:
You can combine the two to `SERVER_URL=localhost:8080`. so for production it might be `SERVER_URL=https://www.url.com`

Reply:
fix